setup.py

```diff
@@ -49,7 +49,7 @@ def get_version():
     ],
     extras_require={
         'test': [
-            'pytest',
+            'pytest==5.3.5',
```

Comment:
Can you please add a comment of why we're pinning this

Reply:
added a comment explaining the reason